pkg/splunk/controller/util.go

```diff
@@ -113,6 +113,15 @@ func MergePodSpecUpdates(current *corev1.PodSpec, revised *corev1.PodSpec, name
 		result = true
 	}
 
+	// Check for changes in Init containers
+	if len(current.InitContainers) != len(revised.InitContainers) {
```

Comment:
Nit: could we have used `reflect` API here to check for changes?

Reply:
+1